ios/FluentUI/Notification/NotificationView.swift

```diff
@@ -455,14 +341,15 @@ open class NotificationView: UIView {
 
     open override func didMoveToWindow() {
         super.didMoveToWindow()
+        tokens.didChangeAppearanceProxy()
```

Comment:
We call the updateForCurrentTheme() as moving to a new window does not necessarily mean that the theme changed.